src/components/ui/searchIcon.tsx

```diff
@@ -0,0 +1,19 @@
+export function SearchIcon(props: any) {
```

Comment:
Pretty sure we don't need that. In components, you can see how I do Check and Pencil. Maybe there is something similar for serach